### PR TITLE
Adjust current page between table stories

### PR DESCRIPTION
**Pull request by edleeks87**
# Description
Fixes defect where pageSize caused incorrect currentPage value

### CODE REVIEW

File: src/components/table/table.stories.js [hunk 1]
```diff
@@ -112,8 +103,15 @@ const buildRows = ({ pageSize, totalRecords }) => {
...
+const roundPageValue = (value) => {
+  const roundedValue = Math.round(value);
+  if (!value || roundedValue < 1) return '1';
+  return roundedValue;
+};
+
 const setCurrentPage = ({ totalRecords, pageSize, paginate }) => {
-  const maxValidPage = Math.round(totalRecords / pageSize) + (totalRecords % pageSize);
+  const roundedValue = roundPageValue(totalRecords / pageSize);
+  const maxValidPage = roundedValue < totalRecords / pageSize ? roundedValue + 1 : roundedValue;
```
Comment (chrisbarber86) [inline src/components/table/table.stories.js]: Might not an idea to put some brackets into this line to make it a bit clearer what is happening.

Comment (chrisbarber86) [reply src/components/table/table.stories.js]: Total pages could be expressed as follows:

```
Math.max(Math.ceil(totalRecords / pageSize), 1)
```
you would want to check that `pageSize` is set and if it's not then return `1`.

Comment (grabkowski) [reply src/components/table/table.stories.js]: or create a const containing `totalRecords / pageSize`

Comment (edleeks87) [reply src/components/table/table.stories.js]: I went with a mix of both!

File: src/components/table/table.stories.js [hunk 2]
```diff
@@ -185,7 +176,15 @@ storiesOf('Table', module)
       store.set({ currentPage: setCurrentPage(props) });
       props.size = select('size', OptionsHelper.tableSizes, Table.defaultProps.size);
       props.isZebra = boolean('zebra striping', false);
-
+      props.theme = select(
+        'theme',
+        [
```
Comment (stevebutler2210) [inline src/components/table/table.stories.js]: Is there a reason you can't just pass in the whole `OptionsHelper.tableThemes` object?

Comment (edleeks87) [reply src/components/table/table.stories.js]: Yeah haha no idea why I've done that here, the classic story only has two variants but the DLS has them all 🤦‍♂

**Pull request merged**